### PR TITLE
Improve rest connection

### PR DESCRIPTION
**Pull request by PedroRossa98**
Change from request lib to the http.client you will need to run:

```
pip install python-http-client
```

### CODE REVIEW

**Comment by PedroRossa98** [comment]
@bgeneto  Can you test if this works in one of your pendulums, and if it is stable?

**Comment by bgeneto** [comment]
> @bgeneto Can you test if this works in one of your pendulums, and if it is stable?

I've been testing this branch which replaces the recommended Requests package (higher-level HTTP client interface) with the `http.client` low level http module. I had use the class `http.client.HTTPSConnection` instead, because we are behind a secure reverse proxy. So I had to add an `if` to the code... 

After two days I could say that there is no change in stability, only a little speed improvement in the connection because now you are doing only one DNS lookup with the `http.client` instead of several with `requests`.  Also, the Requests module do so much behind the scenes, so it is naturally (milliseconds) slower. 

IMHO, the way we should resolve connection problems is to use the timeout properly in every request and do exception handling accordingly. The python script (main.py) should not run indefinitely once a connection problem is detected. We should restart the main script (via cronjob) in those cases. That's what I am proposing in the `UnB` branch and that's the only way I found to keep the secondary pendulums online with minimal downtime (1 or 2 minutes) along several days.

In short, I don't support the change from `requests` to `http.client` because there is no real benefit in doing so. I think it would be better if we could test the `UnB` branch on your side (along with the provided cronjob) and see if we can merge the branch. Regards.
